CONTRIBUTING.md

```diff
@@ -82,14 +82,23 @@ pyenv install 3.10
 
 ```powershell
 C:\python27\python -m virtualenv wvenv27
-wvenv\Scripts\activate
+wvenv27\Scripts\activate
 python setup.py develop
 # I noticed running the above step was failing, but it appears it successfully installs the client:
 # error: Couldn't find a setup script in c:\users\zhaberma\appdata\local\temp\easy_install-m3rrqp\importlib_metadata-8.5.0.tar.gz
 # And after running the below commands, then rerunning the above command, the error does not reappear
 
-pip install pytest pytest-cov
+pip install pytest pytest-cov 
 pytest tests/py27
+pip install pylint==1.9.5 # Last version supporting Python 2.7
+pylint .\src\py23client\v27
+pip install isort==4.3.21 # Last version supporting Python 2.7
+isort -rc .\src\py23client\v27
+pip install autopep8==1.5.7
```

Comment:
We should not use autopep8, but should try to use black.

Otherwise this is good.